resources/views/user/details.blade.php

```diff
@@ -0,0 +1,334 @@
+@extends("layouts.app")
+
+@section("meta")
+	<meta name="csrf-token" content="{{ csrf_token() }}">
+@endsection
+
+@section("title", $user->name)
+
+@section("content")
+	<div class="padded content student profile">
+		<div class="ui stackable grid">
+			<div class="four wide column user-basic-info">
+				<!-- User avatar -->
+				<img class="ui small circular image user-avatar" src="<?php echo asset($user->picture_url) ?>"/>
+				<!-- User name -->
+				<h2 class="center" style="text-align: center;">{{ $user->name }}</h2>
+				<!-- Contact button -->
+				<a class="fluid ui primary button buttonSpace" href="mailto:{{ $user->email }}" style="margin-bottom: 10px;"><i class="icon envelope"></i>Contact</a>
+				<!-- Edit button -->
+				@if ($user->id == Auth::user()->id)
+					<a class="fluid ui grey button buttonSpace" href="#" style="margin-bottom: 10px;">Edit</a> <!--TODO: Falta cambiar el href-->
+				@endif
+				<!-- Skillset pills -->
+				@if (count($user->skillset) > 0)
+					<div class="ui left aligned detail-container">
+						<p><strong>Skillset</strong></p>
+						@foreach($user->skillset as $skill)
+							<div class="ui label pill">{{ $skill->name }}</div>
+						@endforeach
+					</div>
+				@endif
+			</div>
+			<div class="twelve wide column">
+				<div class="ui top attached tabular menu">
+					<a class="active item" data-tab="projects">Projects</a>
+					<a class="item" data-tab="second">Second</a>
+					<a class="item" data-tab="courses">Courses</a>
+				</div>
+				<div class="ui bottom attached active tab segment" data-tab="projects">
+					@if(isset($user->projects) && count($user->projects) > 0)
+						<div class="ui two column stackable grid">
+							@foreach ($user->projects as $project)
+								@projectCard(['id'=> $project->id,'projectImage' => $project->photo, 'projectName' => $project->name, 'projectShortDescription' => $project->short_description, 'labels' => $project->labels, 'publicMilestone' => 'shipping'])
+								@endprojectCard
+							@endforeach
+						</div>
+					@else
+						<div id="no-projects-msg" class="ui message">
+							<div class="header">
+								No projects found!
+							</div>
+							<p>Looks like this user is not participating in any projects... yet!</p>
+						</div>
+					@endif
+				</div>
+				<div class="ui bottom attached tab segment" data-tab="second">
+					Second
```

Comment:
Is this supposed to be like this?

Reply:
Yep. es parte de los missing que vienen en la description del PR pero le pondré un TODO comment como quiera :) thanks!